gns3/pages/server_preferences_page.py

```diff
@@ -63,9 +63,8 @@ def __init__(self, parent=None):
 
         # load all available addresses
         for address in QtNetwork.QNetworkInterface.allAddresses():
-            if address.protocol() == QtNetwork.QAbstractSocket.IPv4Protocol:
```

Comment:
Shouldn't we do something like address.protocol() == QtNetwork.QAbstractSocket.IPv4Protocol or address.protocol() == QtNetwork.QAbstractSocket.IPv6Protocol

I am just afraid we end up with exotic addresses ;)